pyproject.toml

```diff
@@ -24,6 +24,7 @@ dependencies = [
     "threadpoolctl>=3.0.0",
     "tqdm",
     "zarr>=0.2.16",
+    "xarray",
```

Comment:
Honestly I would prefer to avoid this.
We can add it for doc building.

Reply:
Done. Removed from core. added back to full and to docs.